schema.yaml

```diff
@@ -16,6 +16,11 @@ x-google-marketplace:
           type: REPO_WITH_REGISTRY
         image.tag:
           type: TAG
+    ubbagent:
+      properties:
+        ubbagentImage:
+          type: FULL
+          default: "gcr.io/cloud-marketplace-tools/metering/ubbagent:latest"
```

Comment:
can we/should we be using a pinned version?

Reply:
Good point. I'm going to make an update on this, there is a marketplace practice I've seen often where we serve the sidecar image from our repo, thus effectively pinning it.